release-notes.txt

```diff
@@ -1,3 +1,23 @@
+OpenMDAO Version 1.7.1 Alpha Release Notes
+July 18, 2016
+
+Features:
+* Add an `xtol` option to Newton. This ceases iteration when the Newton step shrinks below a threshold.
+* Better user_controllable solver convergence printing (iprint).
+* Added a SAND example from our users into our Examples section in our documentation. 
+* Implemented better error handling when user gives too many args to connect.
+* Added `solve_linear` method to the LinearSystem component.
```

Comment:
Modified `linear_system` so that its derivatives are solved with an additional LU back-substitution solve_linear.